playground/csp/vite.config.js

```diff
@@ -35,6 +35,9 @@ const createMiddleware = (file, transform) => async (req, res) => {
 }
 
 export default defineConfig({
+  html: {
+    cspNonce: noncePlaceholder,
+  },
```

Comment:
```suggestion
```
This is already set below in line 48.

Reply:
I was looking for that, but somehow didn't see that. Fixed.